packages/trader/src/Modules/CFD/Containers/props.types.ts

```diff
@@ -16,3 +19,20 @@ export type TPasswordResetAndTradingPasswordManager = {
     platform: 'mt5' | 'dxtrade';
     account_group: 'real' | 'demo';
 };
+
+export type TResetPasswordIntent = {
+    current_list: { [key: string]: DetailsOfEachMT5Loginid };
+    children({ ...props }): React.ReactElement;
+    is_eu: boolean;
+};
+
+export type TCFDResetPasswordModal = {
+    current_list: { [key: string]: DetailsOfEachMT5Loginid };
```

Comment:
according to the current_list() getter in cfd-store.js, `current_list` is not an object but an array with string indexes instead of numbered indexes. What do you think? @hirad-rewok @balakrishna-binary @matin-binary

Reply:
Such a nice suggestion. It works well in the code.